docs/xgcpolicy.md

```diff
@@ -104,10 +104,10 @@ The following options can also be specified on the command line with `-Xgcpolicy
 The behavior of the following options is different when specified with `-Xgcpolicy:balanced`:
 
 `-Xcompactgc`
-: Compaction occurs when a System.gc() call is received (default). Compaction always occurs on all other collection types.
+: Compaction occurs when a `System.gc()` call is received (default). Compaction always occurs on all other collection types.
```

Comment:
I don't think this has anything to do with System GC.

It's as simple as that global (mark-sweep) GC will have mandatory compact operation (which otherwise is just optional triggered by various internal heuristics.

It's irrelevant global GC is triggered explictly (System GC) or implicitely, if it's concurrnet or STW global etc.

Reply:
Sorry, not sure what this should say instead

Reply:
A simple examples:
Compaction is forced in each Global GC cycle 
or in active:
(The option) Forces compaction in each Global GC cycle.

and for nocompact (in passive):
Internal compaction heuristics in Global GC cycle are disabled.

Reply:
Updates made.